napari/layers/base/base.py

```diff
@@ -536,6 +542,13 @@ def block_update_properties(self):
         yield
         self._update_properties = True
 
+    def set_view_slice(self):
```

Comment:
do we really want to make this public?

Reply:
we could call this something like `refresh` instead and leave `_set_view_slice`. I think it might be nice for plugin developers to have this method available to them

Reply:
ok, yeah I think `refresh` could work, since I think that developers might be confused as to what `set_view_slice` means :)

Reply:
done